src/s1reader/s1_orbit.py

```diff
@@ -255,6 +257,53 @@ def get_orbit_file_from_dir(zip_path: str, orbit_dir: str, auto_download: bool =
 
     # search for orbit file
     orbit_file_list = glob.glob(os.path.join(orbit_dir, 'S1*.EOF'))
+
+    orbit_file = get_orbit_file_from_list(zip_path, orbit_file_list)
+
+    if orbit_file:
+        return orbit_file
+
+    if not auto_download:
+        msg = (f'No orbit file was found for {os.path.basename(zip_path)} '
+                f'from the directory provided: {orbit_dir}')
+        warnings.warn(msg)
+        return
+
+    # Attempt auto download
+    orbit_file = download_orbit(zip_path, orbit_dir)
+    return orbit_file
+
+
+
+def get_orbit_file_from_list(zip_path: str, orbit_file_list: list) -> str:
```

Comment:
Are the changes related to `s1_orbit.py` actually caused by a merge with develop?

Reply:
Yes, especially the PR #79 to make the RTC workflow to work again.